scripts/publish

```diff
@@ -15,11 +15,18 @@ if ! command -v "${PREFIX}twine" &>/dev/null ; then
     exit 1
 fi
 
+if ! command -v "${PREFIX}mkdocs" &>/dev/null ; then
+    echo "Unable to find the 'mkdocs' command."
+    echo "Install from PyPI, using '${PREFIX}pip install mkdocs'."
+    exit 1
+fi
```

Comment:
Should we add a `dev-requirements.txt` that builds on top of `test-requirements.txt` and adds `mkdocs` and `twine`?

Reply:
Ah, good point. *Or*… we could add a nox `release` task that installs those for us? 😅 @sethmlarson

Reply:
I tried to do that, and the fact that the `publish` script currently defaults to using `venv/bin/` as a command prefix is there is a `venv` directory means that it can't use the `mkdocs` and `twine` commands we'd install so long as we have a local virtualenv.

Reply:
@florimondmanca That's why I wanted a release task :)

Reply:
> Should we add a `dev-requirements.txt` that builds on top of `test-requirements.txt` and adds `mkdocs` and `twine`?

Personally I would prefer all the `requirements.txt` files to be grouped and match "requirements-*.txt", i.e.
```
requirements-dev.txt
requirements-test.txt
```